ansible-scylla-node/tasks/adjust_keyspace_replication.yml

```diff
@@ -24,9 +24,23 @@
   loop: "{{ groups['scylla'] }}"
   run_once: true
 
+- set_fact:
+    scylla_admin_username: "{{ scylla_admin_default_user }}"
+    scylla_admin_password: "{{ lookup('ini', scylla_admin_default_user, section='cql_credentials', allow_no_value=true, default=omit, file=inventory_file) }}"
+  when: "scylla_admin_default_user is defined and 'cql_credentials' in groups"
+
+- fail:
+    msg: "Make sure that {{ scylla_admin_default_user }} is defined in the 'cql_credentials' section"
+  when: "scylla_admin_default_user is defined and scylla_admin_password is not defined"
```

Comment:
Did you simulate this case to see that it fails as expected?

Reply:
Same question about quotes

Reply:
Yes, I simulated and it fails as expected.

Reply:
quotes removed